fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/Binder.java

```diff
@@ -66,7 +66,8 @@ public Binder(Pattern pattern, GroupExpression groupExpression) {
      */
     public OptExpression next() {
         // For logic scan to physical scan, we only need to match once
-        if (pattern.children().size() == 0 && groupExpressionIndex.get(0) > 0) {
+        if ((!pattern.isPatternMultiJoin() && pattern.children().size() == 0) &&
```

Comment:
does it mean, for MULTI_JOIN, it can match a tree whose node is JOIN but without any children ?